source/installation/install-with-docker.txt

```diff
@@ -0,0 +1,142 @@
+.. _rm-install-docker:
+
+===================
+Install with Docker
+===================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can install and run Relational Migrator as a Docker container. 
+Running Relational Migrator with Docker ensures an installation process 
+that is both independent of the operating system and straightforward 
+to replicate.
+
+About this Task
+---------------
+
+You can also use `Docker Compose <https://docs.docker.com/compose/>`__ 
+to run Relational Migrator. You can find a Docker Compose example file
+in the 
+`download center <https://www.mongodb.com/try/download/relational-migrator>`__.
+
+.. #TODO# Update the link, when the compose file is added. 
+
+Before you Begin
+----------------
+
+Install `Docker <https://docs.docker.com/install/>`__
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Pull the MongoDB Docker Image
+      
+      .. code-block:: sh
+
+         docker pull public.ecr.aws/v4d7k6c9/relational-migrator:latest
+
+   .. step:: Run the Image as a Container
+      
+      The tabs below show various docker commands that are useful for 
+      running Relational Migrator:
+
+      .. tabs::
```

Comment:
I don't have a strong opinion on this, but I found the user of tabs here a little unexpected. Usually we use tabs when there are several alternative ways of accomplishing a task. In this case we are using them to describe different features that are available. Happy to keep this if you feel it's the best way of explaining the options, but it feels a bit unusual.

Reply:
I think this is the best path forward, the reason being:
1. Its possible that 90% of people will use the `Basic` option and not need the additional overhead of knowing / learning / understanding the volume mounting. 
2. This keeps the task shorter for the common user but also has the option to expand to the more complicated feature based use case with the other tabs.